bokeh/transforms/image_downsample.py

```diff
@@ -11,20 +11,21 @@
 
 
 def source(**kwargs):
-  kwargs['transform'] = {'resample':'heatmap',
-                         'global_x_range' : [0, 10],
-                         'global_y_range' : [0, 10],
-                         'global_offset_x' : [0],
-                         'global_offset_y' : [0],
-                         'type' : 'ndarray'
-
-  }
-  kwargs['data'] = {'x': [0],
-                    'y': [0],
-                    'dw' : [10],
-                    'dh' : [10],
-                   }
-  return ServerDataSource(**kwargs)
+    kwargs['transform'] = {'resample':'heatmap',
+                           'global_x_range' : [0, 10],
+                           'global_y_range' : [0, 10],
+                           'global_offset_x' : [0],
+                           'global_offset_y' : [0],
+                           'type' : 'ndarray'
```

Comment:
hehe, I also prefer that, but I suppose I'm now trained to use the style that pyflakes wants me to use. I shall change it.